Added mobile view for History

jaerlcruz opened the pull request.

jaerlcruz committed:
Update History.js

MekhiHart commented on client-sq/src/components/History.js:
```diff
@@ -55,8 +66,141 @@ const History = ({ theme }) => {
-
-
+    <div>
+    { isMobile ? (
```
This might be rendering the wrong components. The text enlarges when the device is in mobile view

MekhiHart reviewed (changes requested):
UI breaks in mobile for history layout

jaerlcruz committed:
Update History.js

jaerlcruz committed:
Update History.js

jaerlcruz committed:
Update History.js

MekhiHart reviewed (changes requested):
Magnifying glass also break in mobile view

<img width="379" alt="image" src="https://github.com/user-attachments/assets/0f8edf0f-d90b-4c1e-b0ae-9a3fb7159a58">

jaerlcruz committed:
Update History.js

jaerlcruz committed:
Update History.js

jaerlcruz committed:
Modified text sizes for History

jaerlcruz committed:
Update History.js

removed text size change based on screen size; slightly adjusted padding/margin specifications
